dashboard/lib/services/lesson_import_helper.rb

```diff
@@ -6,6 +6,25 @@ def self.update_lockable_lesson(script_levels, lesson_id)
     [create_activity_with_levels(script_levels, lesson_id, 1)]
   end
 
+  def self.create_lesson_resources(cb_resources, course_version_id)
+    cb_resources.map do |cb_resource|
+      raise unless cb_resource['slug']
+      resource = Resource.find_or_initialize_by(
+        course_version_id: course_version_id,
+        key: cb_resource['slug']
+      )
+      resource.name = cb_resource['name']
+      resource.url = cb_resource['url']
+      resource.type = cb_resource['type']
+      resource.audience = cb_resource['student'] ? 'Student' : 'Teacher'
+      resource.assessment = false
```

Comment:
should assessment ever be set to true when importing existing resources from CB?

Reply:
@tess323 @dmcavoy I'm going to go ahead and merge this PR, but would like any background info you may have on when a Resource might be marked as an assessment in our new system, and if there's anything related to importing assessment resources for either January or April.